### PR TITLE
Fixes 170

### PR DESCRIPTION
**Pull request by Semicolon1071**
Issue #170 describes the requirements for obtaining consent from the data-subject, along with the document that is the consent itself.

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw609f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw610f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

File: docs/development/security/consent.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+* Almindelige personoplysninger
+
+## 4. Hvor dine personoplysninger stammer fra
+
+Personoplysningerne stammer fra registreringen af brugeren i applikationen.
+
+## 5. Opbevaring af dine personoplysninger
+
+Personlige oplysninger slettes 1 år efter, at brugeren er erklæret inaktiv, hvilket afhængigt af institutionen enten kan være et aktivt valg foretaget af institutionen eller som resultat af manglende anvendelse af systemet.
```
Comment (frederikspang) [inline docs/development/security/consent.md:40]: Does it, though?

Comment (frederikspang) [reply docs/development/security/consent.md:40]: Hvis ikke - Så skal vi nok lave et issue på at få implementeret dette.

Comment (Torpstitution) [reply docs/development/security/consent.md:40]: Ja, det er der også en issue på. Og i tilfælde af at der findes en anden løsning, så skal det her selvfølgeligt ændres. Det er ligeså meget et bud på hvordan man kunne gøre, fordi at vi har diskuteret det, også med Ulrik, men implementationen mangler stadigt. De her dokumenter skal generelt revideres så snart vi tilføjer behandling af personlig information eller ændre organisationsstruktur, derfor er det svært at lave en helt korrekt version før at det rent faktisk skal udbydes.

**Review by frederikspang — APPROVED** [review]

**Review by Markusjeppesen — APPROVED** [review]

**Pull request merged**